### PR TITLE
update USWDS and add form and input support

### PR DESCRIPTION
**Pull request by emilyville**
I updated the assets from USWDS to the latest version. 

There may be some images missing for form elements - still need to go over this in more detail. 

Don't merge yet, but wanted to generate a shareable link with the PR :)

### CODE REVIEW

File: _sass/components/_accordions.scss [hunk 1]
```diff
@@ -28,16 +27,16 @@ $accordion-border: 3px solid $color-gray-lightest;
   }
 
   button[aria-expanded=false] {
-    background-image: url('../images/icons/plus.png');
-    background-image: url('../images/icons/plus.svg');
+    background-image: url('../img/plus.png');
+    background-image: url('../img/plus.svg');
```
Comment (akainic) [inline _sass/components/_accordions.scss]: I think we may need to change this file path back to `images/icons/` for our repo. We have these images in that directory now, so the incorrect file path may be why the images appeared to be missing.

File: _sass/components/_alerts.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+.usa-alert-text {
+  font-family: $font-sans;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+.usa-alert-success {
+  background-color: $color-green-lightest;
+  background-image: url('../img/alerts/success.png');  
+  background-image: url('../img/alerts/success.svg');
```
Comment (akainic) [inline _sass/components/_alerts.scss]: I think these images are currently missing

File: _sass/core/_defaults.scss [hunk 3]
```diff
@@ -3,7 +3,7 @@
 // These variables can be modified in _variables.scss
 
 // Typography
-$em-base:             16px; // default is already defined in bourbon
+$em-base:             10px; // default is already defined in bourbon
```
Comment (akainic) [inline _sass/core/_defaults.scss]: @emilyville Maybe why everything looks bigger? I wonder why they made this change without changing all the other sizes, seems pretty drastic

**Comment by emilyville** [comment]
closing because this is happening in #1153

**Pull request closed without merging**